### DISCUSSION TITLE
I followed all the installation steps but the filter-list isn't blocking anything at all.

### DISCUSSION THREAD

**Discussion by MammaGrus** [Lennolium/PrivacyShield#24] [Questions and Answers] [answered]
I have turned off every single other filter to check and installed the list twice with the same results. I am using the diamond list. I tested my browser, Safari with Adguard for Mac, on BlockAds! and got a score of 9%

**Comment by Lennolium** [comment]
Hello @MammaGrus!

Thanks for using PrivacyShield, really appreciated.

Did you test your blocking performance with these two websites: [D3Ward](https://d3ward.github.io/toolz/adblock.html), [BlockAds](https://blockads.fivefilters.org/)? If you tell me your results, I can further investigate your problem. Which adblocker are you running (AdGuard, AdGuard Home, Pi-Hole, Little Snitch, ...)?

Also keep in mind, that the diamond version is not updated to v2.0.0, so the domains are maybe too old. While I'm still in the updating process of all lists to 2.0.0, I would recommend you switch to the [platinum](https://raw.githubusercontent.com/Lennolium/PrivacyShield/master/Filters/PrivacyShield-platinum.txt) version, which I already updated. Please try this list and run the website tests (see two links above) again and give me the results.

Cheers

**Comment by MammaGrus** [comment]
Yes I did. It's obvous too as youtube ads are now showing. I shall try the platinum one tho
￼￼

--
Mvh
Isak Hirseland

**Reply by Lennolium** [reply]
Alright. Regarding YouTube ads, it is almost impossible for adGuard/host filter list to block them, as YouTube changed the way they serve ads. Mostly browser plugins are the only way to effectively block ads on youtube.

**Reply by MammaGrus** [reply]
Maybe it's me doing something wrong with the installation because the instructions were somewhat unclear in my opinion. Could you walk me through all the steps in a bit more detail perhaps?

**Reply by Lennolium** [reply]
Yes of course I can help you. But first tell me which platform you are on (Windows, macOS, iOS, ...) and which adblocker (AdGuard, AdGuard Home, Pi-Hole, ...) you are using.

**Reply by MammaGrus** [reply]
Thanks
macOS adguard for Mac

**Reply by Lennolium** [reply, marked as answer]
Ok alright.

1. Check that you installed the macOS app following the steps outlined [here](https://adguard.com/kb/adguard-for-mac/installation/) and it should automatically install the browser plugin for your default browser as written [here](https://adguard.com/kb/adguard-for-mac/browser-assistant/). Check if the browser plugin is activated in your browser and the adguard icon is appearing somewhere.
2. Go to `Preferences` -> open the tab `Filters` -> click on the `+` button on the **lower left** side (see screenshot attached) 
![filters](https://github.com/Lennolium/PrivacyShield/assets/76521981/835116c4-30a2-45e2-af3b-f72b89d7e385)
3.  Give the new filter entry a `Name`, for example "PrivacyShield Platinum" and paste the [link](https://raw.githubusercontent.com/Lennolium/PrivacyShield/master/Filters/PrivacyShield-platinum.txt) (right-click here -> copy link) from the filter into the `URL/Link` field.
4. Save the entry and check that it is enabled (blue checkmark next to the newly created entry with the chosen name).
5. Restart the app (or to be sure the mac).

Now try to run the tests again and tell me the results.
Hope this helps!
